salesagility/SuiteCRM
# Fix duplication of email address fields.

## Description



This change stops the duplication of email addresses in non standard module list views.

## How To Test This

One example can be found by:
1) Create an email campaign,
2) Have the email campaign sent,
3) Navigate to the email queue (admin panel, email queue),
4) view the email address field and ensure that there is only one email address in each field.

Extra checks:
1) Change the opt in settings in the admin panel (Admin panel, email settings, Opt in settings),
2) try all three and each time the new setting is saved, complete the steps above to ensure that the field still has a single entry.

## Types of changes


### Final checklist
